tests/unit/SidebarFilterTest.js

```diff
@@ -253,6 +258,7 @@ describe('Sidebar', () => {
             );
         });
 
+        // NOTE: This is also for #focus mode, should we move this test block?
```

Comment:
I am not sure what the original intention here was. Maybe @tgolen can remember. This comment stuck out to me as pretty confusing and something we should fix.

**context:** Working on a PR and encountered a failing test with this title:

>Sidebar › in default (most recent) mode › all combinations of isArchived, isUserCreatedPolicyRoom, hasAddWorkspaceError, isUnread, isPinned, hasDraft › the booleans true,false,false,true,false,false

But when I look at the lines of code it's referencing I see that we are not testing "most recent" but "focus" mode.

Reply:
This test also confuses me if the intention is that we should be testing "Most recent" mode. 

If we are trying to see that a report is in the "Most recent" list then most all reports will be in the list.

Or at least doesn't seem dependent on the arguments we are passing.

>isArchived, isUserCreatedPolicyRoom, hasAddWorkspaceError, isUnread, isPinned, hasDraft

All of those will would show in "Most recent" (all other things about the report being "normal")

Reply:
Ok after reviewing the two blocks - I have determined they are identical and we should be able to remove this...

I don't think this was intended to be put here and it's not logical to test.